### ISSUE TITLE
KeepShare doesn't save changes to Onedrive

### ISSUE THREAD

**Issue by sanddk** [pfn/keepshare#47]
I have my database on Onedrive and my key file on Google drive. But I've noticed, that changes made on the phone doesn't get synced to Onedrive and then to my PC. I have added the database through KeepShare using the OneDrive option. It syncs from the computer to the phone, but seemingly not the other way, as I can see that the file in Onedrive hasn't changed after adding an entry on the phone.

**Comment by pfn** [comment]
Will have to verify this. Changes are pushed to OneDrive via SAF whenever
updates occur. Does the file appear updated when opening the onedrive app
on the phone after modifying from keepshare?

**Comment by sanddk** [comment]
No, the file is not updated when browsing to it using the onedrive app.
Could it be 2 factor login on my office365 account?

**Comment by pfn** [comment]
I'm not aware of any reason why it shouldn't. Perhaps there's a setting in
one drive that prevents/enables outside apps to update its data.

**Comment by sanddk** [comment]
Cannot see anysettings in the onedrive app that should prevent it from syncing. Should I just try to clear all app data, and set it up again?

**Comment by sanddk** [comment]
I have just abandoned onedrive, and use another cloud service instead. I think that the issue might be related to 2 factor login to onedrive, but not sure. I have experienced a number of other issues pertaining to my 2 factor login related to different microsoft services...
Anyway, after moving out from onedrive, sync works as intended...

**Comment by pfn** [comment]
Thanks for the feedback
